src/coreclr/tools/r2rtest/BuildOptions.cs

```diff
@@ -72,7 +72,7 @@ public string CoreRootOutputPath(CompilerIndex index, bool isFramework)
         /// <summary>
         /// Creates compiler runner instances for each supported compiler based on the populated BuildOptions.
         /// </summary>
-        /// <param name="isFramework">True if compiling the CoreFX framework assemblies</param>
+        /// <param name="isFramework">True if compiling the runtime libraries assemblies</param>
```

Comment:
We've typically used "core libraries" (analogous to corelib which is just one). I've not heard "runtime libraries ". I edited the glossary too Core libraries

Reply:
> I've not heard "runtime libraries "

* https://github.com/search?q=repo%3Adotnet%2Fruntime%20%22runtime%20libraries%22&type=code
* https://learn.microsoft.com/en-us/dotnet/standard/runtime-libraries-overview

CoreLib specifically implies `System.Private.CoreLib`.